servicebroker/lib/util/apiServerUtil.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+var request = require('request');
+
+module.exports = function(apiServerUri){
+  var apiUtil = {};
+  apiServerUri = apiServerUri.replace(/\/$/g, "").toLowerCase();
```

Comment:
Familiar with dbUri , the provided api server uri may be "http://autoscaler.boshlite.com" or "http://autoscaler.boshlite.com/". So we need to remove the last "/".